### DISCUSSION TITLE
Settings to dump TG-16/PCE, Game Gear, SMS games

### DISCUSSION THREAD

**Discussion by RascalDungeon** [sanni/cartreader#946] [Need help]
Title says it all. I looked through here and the Wiki and tried searching the web, but couldn't figure out what settings to use to dump these carts (Voltage, CLK0, etc.). All I see is info for NES, SNES, etc.

**Comment by PsyK0p4T** [comment]
For all of these, set voltage to 5V, and all other switches to off.

If you aren't sure about the voltage to select for a system, you can check directly into the code for the following lines : 
setVoltage(VOLTS_SET_3V3) or setVoltage(VOLTS_SET_5V)

**Reply by RascalDungeon** [reply]
Thanks.

So there is no option to dump PC Engine or Turbografx-16 games. How do I add it?

**Reply by PsyK0p4T** [reply]
https://github.com/sanni/cartreader/wiki/How-to-flash-the-Arduino 
Look at step 7 ;)

**Reply by RascalDungeon** [reply]
I was going to update the firmware, but I am stuck at step 2 of that page.  

I don't understand what it means by "Install CH341 drivers if needed by running SETUP.exe in the "CH341 Drivers" folder."

**Reply by RascalDungeon** [reply]
Ok, figured out step 2. But how do I know what hardware version I have?

**Reply by Ancyker** [reply]
Post a picture.

**Reply by RascalDungeon** [reply]
I figured it out. Thanks for pointing me to the right page. Every few months I go through this. I always just search for a manual, which takes me here:

https://github.com/sanni/cartreader/wiki/User-Manual

Which isn't very useful. So I end up asking here. 

Now I have the right page bookmarked:

https://github.com/sanni/cartreader/wiki/How-to-flash-the-Arduino

There should be a link from the manual page.
